release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -487,7 +487,7 @@ The assisted installer and installer-provisioned installation for bare metal clu
 [id="ocp-4-8-configure-vips-to-run-on-control-plane"]
 ==== Configure network components to run on the control plane
 
-In {product-title} 4.8, for installer-provisioned installations, you must configure the `apiVIP` and `ingressVIP` virtual IP addresses to run exclusively on the control plane nodes. By default, {product-title} allows any node in the machine configuration pool to host the `apiVIP` and `ingressVIP` virtual IP addresses. Since many environments deploy worker nodes in separate subnets from the control plane nodes, configuring the `apiVIP` and `ingressVIP` virtual IP addresses to run exclusively on the control plane nodes prevents issues from arising due to deploying worker nodes in separate subnets.
+If you need the virtual IP (VIP) addresses to run on the control plane nodes in a bare metal installation, you must configure the `apiVIP` and `ingressVIP` VIP addresses to run exclusively on the control plane nodes. By default, {product-title} allows any node in the worker machine configuration pool to host the `apiVIP` and `ingressVIP` VIP addresses. Since many bare metal environments deploy worker nodes in separate subnets from the control plane nodes, configuring the `apiVIP` and `ingressVIP` virtual IP addresses to run exclusively on the control plane nodes prevents issues from arising due to deploying worker nodes in separate subnets.
```

Comment:
One lingo-ism: please change "Since" to "Because".  See ISG, around p401.

I suppose it isn't required, but the convention tends to be that each enhancement includes an xref to more info.  If there is more information, please add the xref.  I admit that I don't know the subject area, but I'm also not clear about how I'd configure these VIPs.  It could just be me.

Reply:
I keep getting mixed messages about xrefs. I thought we weren't allowed to put in xrefs, then we were, then we weren't again. I'm a bit confused.